src/gui/addnewtorrentdialog.h

```diff
@@ -38,6 +38,7 @@
 
 #include "base/bittorrent/infohash.h"
 #include "base/bittorrent/torrentinfo.h"
+#include "base/bittorrent/session.h"
```

Comment:
Now it does not make any sense since AddTorrentParams class is in `base/bittorrent/addtorrentparams.h`